Makefile

```diff
@@ -94,7 +94,16 @@ check-go-format: prebuild-check
 	&& exit 1 \
 	|| true
 
-
+.PHONY: install-git-hooks
+## Creates useful git hooks (e.g. pre-push go-format-code check)
+install-git-hooks:
+	ln -sfv ../../.pre-push .git/hooks/pre-push
```

Comment:
The rename of a repo is something that doesn't happen that often, at least in general :).